### ISSUE TITLE
Emphasize(), SpeechPitch() and Transmit() functions do not work properly

### ISSUE THREAD

**Issue by Arcprotorp** [EDCD/EDDI#571]
## EDDI version in which issue found
v.3.0.1-b2 standalone

## Steps to reproduce

0. Using Win10
1. Creating a new test script in SpeechResponder, to try out personalisation options
2. Trying out various Cottle functions like Emphasize(), SpeechPitch(), and Transmit()
3. Testing it by clicking on "Test button", using various secondary arguments in Emphasize and SpeedPitch, by provided in "Help"

## Expected
I expected the exact events described in "Help"

## Observed
1. Emphasize and SpeedPitch does not work with any argument. Reads out as normal text.
2. Transmit and it contained argument executes before any other text-reading, although it is on the end of the script. After, every other reading executes without problem. Like I wrote the {Transmit("....")} line of the beginning of the script and not on the end

## Investigation
No further investigation was done, as for previously described functions I found no source any other than EDDI documentation. Not even in Cottle documentations.

**Comment by Tkael** [comment]
These functions are sensitive to the voice that you are using (not all voices are equally compatible). Have you tried these functions with more than one voice?

**Comment by Arcprotorp** [comment]
Thank you! It turns out Zira is not compatible with these functions. Although one question, from a slightly different topic: Can I make nested functions like placing a OneOf() into the argument of Occasionally(). This also led to failure, and the reason seems not to be my incompetency, rather being not familiar with the language.

Regardless, thanks for the suggestion, it solved my original problem.

**Comment by Tkael** [comment]
Yes you can, though you may need to add the escape character `\` in front of any inside quotes.
Example: `{Occasionally(2, "You {OneOf(\"have enough free space to carry\", \"can carry\")} another 332 tonnes")}`

**Comment by Tkael** [comment]
Glad I was able to help. :-)

**Comment by Tkael** [comment]
Re-opening to fix observation 2... that's not the intended behavior. ;-)
